File: MedExpert.Web/ViewModels/Analysis/CommentModel.cs
namespace MedExpert.Web.ViewModels.Analysis
{
    public class CommentModel
    {
        public int SymptomId { get; set; }
        
        public int SpecialistId { get; set; }

        public string Name { get; set; }
        
        public CommentType Type { get; set; }

        public string Text { get; set; }
    }

    public enum CommentType
    {
        Symptom = 1,
        MatchedIndicator = 2,
        RecommendedForAnalysisIndicator = 3
    }
}

File: MedExpert.Web/ClientApp/src/app/store/model/comment.model.ts
export interface IComment {
  symptomId: number;
  specialistId: number;
  symptomName: string;
  name: string;
  type: CommentType;
  text: string;
}

export enum CommentType {
  Symptom = 1,
  MatchedIndicator = 2,
  RecommendedForAnalysisIndicator = 3
}
